ts_adaptive_ashrae.json

```diff
@@ -0,0 +1,158 @@
+{
+  "information": {
+    "summary": "This file contains a series of validation tables for adaptive_ashrae() in pythermalcomfort, jsthermalcomfort, calcAdaptiveASHRAE() in R",
+    "version": "1.0.0",
+    "license": "MIT"
+  },
+  "tolerance": {
+    "tmp_cmf" : 0.1
+  },
+  "data": [
+    {
+      "source": "test_adaptive_ashrae.py",
+      "inputs": {
+        "tdb": 19.6,
+        "tr": 19.6,
+        "t_running_mean": 17,
+        "v": 0.1
+      },
+      "outputs": {
+        "acceptability_80": true
```

Comment:
this output should also contain `tmp_cmf`. I think that the variable that is listed in the tolerance should always be there in the outputs. You can run the function with the above inputs, get the value and add it here to the test